Backend/trainmodel.py

```diff
@@ -0,0 +1,40 @@
+from learner.resnet18_vgg16 import HandwrittenSymbolsClassifier
+from PIL import Image
+import glob
+import torchvision.transforms as transforms
+
+
+model_name = 'resnet' # or 'vgg'
+
+classifier = HandwrittenSymbolsClassifier(root_dir='./learner/datasets/extracted_images/', epochs=5, model_type='resnet')
+
+try:
+    classifier.load_model(f'./learner/models/test_{model_name}.torch')
+    print("Model loaded successfully")
+except Exception as e:
+    try:
+        classifier.train()
+        classifier.save_model("./learner/models/", f"test_{model_name}.torch")
+        classifier.load_model(f'./learner/models/test_{model_name}.torch')
+        print("Model trained and saved successfully")
+    except Exception as e:
+        print(e)
+
+
+
+# testing
+
+transformtosize = transforms.Compose([
+    transforms.Resize((224, 224)),
+    transforms.ToTensor(),
+])
+
+image_paths = glob.glob('tests/testfornew/**/*.png', recursive=True)
+
+for image_path in image_paths:
+    image = Image.open(image_path).convert('RGB')
+    image = transformtosize(image)
+    image = image.unsqueeze(0)
+    prediction = classifier.predict(image)
+    print(f'Prediction for {image_path}: {prediction}')
```

Comment:
Unit tests has scoring system to determine if everything works fine. Suggesting enhancement so that CI/CD integration can be made possible by throwing error incase of a test case failure.

Reply:
I will add around 20-30 realtime test soon, this is just to determine whether even this model works or not

Reply:
No, scoring is necessity. Not acceptable.

Reply:
Okay this could be resolved later. Accepting as it is.